### ISSUE TITLE
PEP440 comparison error (1.0 > 1.0.1)

### ISSUE THREAD

**Issue by ryuusenshi** [royw/Versio#2]
The snippet below produces unexpected behavior:

``` python
v1 = Version('1.0', scheme=Pep440VersionScheme)
v2 = Version('1.0.1', scheme=Pep440VersionScheme)
```

> > > v1 > v2     # returns True

**Comment by royw** [comment]
I'll take a look.  Thank you.

**Comment by royw** [comment]
Fixed for PEP440.  Also added a variable dotted scheme if you need variable number of sequences and not the full PEP440.  Released to pypi as version 0.3.0
